usr.sbin/newsyslog/newsyslog.conf.5

```diff
@@ -117,7 +117,9 @@ This does not consider the current log file.
 .It Ar size
 When the size of the log file reaches
 .Ar size
-in kilobytes, the log file will be trimmed as described above.
+in Kilobytes, or by using any suffix (e.g. K, M, G) supported by
```

Comment:
Sorry to insist, but `kilobyte` should be written as is.
From https://en.wikipedia.org/wiki/Kilobyte:
> The internationally recommended unit symbol for the kilobyte is **kB**.[^1]

Also:
- `mandoc -T lint` will request that `expand_number(3)` is placed before `syslog(3)`, to keep the list sorted alphabetically by manual.
- [`igor(1)`](https://man.freebsd.org/cgi/man.cgi?igor(1)), will recommend to add a comma after _exempli gratia_.

The manual has other minor imperfections but are not related to this pull request. At any rate, a bump in the date should be required.

[^1]: https://en.wikipedia.org/wiki/Kilobyte#cite_note-IEC80000-1

Reply:
Sorry for my poor English,
I would like to make sure you want me to use `kilobyte` instead of `kilobytes`?

Reply:
Sorry for my careless behavior,

![image](https://github.com/user-attachments/assets/1c3ddc7b-aaf8-4642-9bf1-3b66582e5edb)

I think it is what you are trying to say?

Reply:
> [Edited]
> 
> Sorry for my careless behavior,
> 
> ![image](https://private-user-images.githubusercontent.com/182052080/382609814-d73ce231-1669-4c0f-b3d0-3c0ec9dc96c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NzQ0NzgsIm5iZiI6MTczMDY3NDE3OCwicGF0aCI6Ii8xODIwNTIwODAvMzgyNjA5ODE0LWQ3M2NlMjMxLTE2NjktNGMwZi1iM2QwLTNjMGVjOWRjOTZjNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QyMjQ5MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MDRhNzQ2NDM2YTQwNDk2ZWE5MTEzZmE5NzUyMTQ4ZDJlZWM1MWVlODVmNzE4ZGY1YTc0OTgzMDEyNWQ2MTUxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.QUlKPkADUIe-Q49rn_oPD4SqdQnVU3sLbKklqnk31kY)
> 
> I think it is what you are trying to say?

Yes, the letter “k” in kilobyte, should be lowercased (as it currently is).